android/java/org/chromium/chrome/browser/app/domain/KeyringModel.java

```diff
@@ -113,31 +120,48 @@ public void getAccounts(Callbacks.Callback1<AccountInfo[]> callback1) {
         mKeyringService.getAllAccounts(allAccounts -> { callback1.call(allAccounts.accounts); });
     }
 
-    private void addAccountInternal(@CoinType.EnumType int coinType,
-            @KeyringId.EnumType int keyringId, String accountName,
+    public void isWalletCreated(@NonNull final KeyringService.IsWalletCreated_Response callback) {
+        mKeyringService.isWalletCreated(callback);
+    }
+
+    private void addAccountInternal(
+            @CoinType.EnumType int coinType,
+            @KeyringId.EnumType int keyringId,
+            String accountName,
             Callbacks.Callback1<Boolean> callback) {
-        mKeyringService.addAccount(coinType, keyringId, accountName,
-                result -> { handleAddAccountResult(result, callback); });
+        mKeyringService.addAccount(
+                coinType,
+                keyringId,
+                accountName,
+                result -> {
+                    handleAddAccountResult(result, callback);
+                });
     }
 
```

Comment:
This is just change in formatting. Do we have new formatting style? Or it was not properly formatted before? or?

Reply:
Just curious

Reply:
Very odd, right? 
The only explaination I can give is that we have a slightly different formatting style. I didn't pay much attention to this as it's done automatically, but I'll investigate and get back to you.

Reply:
iirc we applied Java formatting ~2 years ago, most likely that file was created before that date. We have it pretty often in old files.

Reply:
That would be great if such formatting rules are forcedly applied to all .java files(maybe iteratively per component in many PRs), so reviewers would not have to review what they don't need to review.

Reply:
I've created an issue for Wallet files: https://github.com/brave/brave-browser/issues/37454